hs/src/Reach/Backend/JS.hs

```diff
@@ -1093,7 +1129,7 @@ jsPIProg cr (PLProg _ _ dli dexports ssm (EPPs {..}) (CPProg _ vi _ devts _)) =
     local (\e -> e {ctxt_maps = dli_maps}) $
       jsViews vi
   mapsp <- jsMaps dli_maps
-  let partMap = flip M.mapWithKey epps_m $ \p _ -> pretty $ bunpack p
+  let partMap = M.foldrWithKey (\ (p, _) _ acc -> M.insert p (pretty $ bunpack p) acc) mempty epps_m
```

Comment:
The paren hate is why I flipped here :)

Reply:
Also, this foldr is really an `M.mapWithKey`. I don't think you should ever do a fold when you mean map

Reply:
I'm using `fold` because I'm changing the keys

Reply:
Yuck